mnemonic-core/src/test/java/org/apache/mnemonic/DurablePersonNGTest.java

```diff
@@ -78,25 +80,74 @@ public boolean reclaim(Long mres, Long sz) {
         }
 
         person = PersonFactory.create(act);
+        int size = rand.nextInt(1024 * 1024) + 1024 * 1024;
+/////////////////////////if size is smaller than remaining act capacity////
+
+        mbh = act.createBuffer(size);
```

Comment:
Please add the second parameter as false to prevent this buffer to be collected. and do the same for other createBuffer()/CreateChunk() calls.